tests/testtext.c

```diff
@@ -46,6 +46,7 @@ static void test_measure_string(void)
 	const WCHAR teststring1[] = { 'M', '\n', '\n', 'M', 0 };
```

Comment:
These should all be `static`. Some compilers will do that. Visual C++ will copy them to the stack.

Reply:
Well, I just followed the code that was already there. We can fix this issue separately.